js/src/components/paid-ads/billing-card/billing-card.js

```diff
@@ -15,20 +16,32 @@ import fallbackBillingUrl from './fallbackBillingUrl';
 import { GOOGLE_ADS_BILLING_STATUS } from '.~/constants';
 import './billing-card.scss';
 
-const { APPROVED } = GOOGLE_ADS_BILLING_STATUS;
+const { APPROVED, PENDING } = GOOGLE_ADS_BILLING_STATUS;
 
 /**
  * Renders a success notice card or a setup card according to the billing status
  * of the connected Google Ads account.
  */
 export default function BillingCard() {
+	const [ showNotice, setShowNotice ] = useState( false );
 	const { billingStatus, hasFinishedResolution } =
 		useGoogleAdsAccountBillingStatus();
 
+	useEffect( () => {
+		// only show notice if we are in pending state before
+		if ( billingStatus.status === PENDING && ! showNotice ) {
+			setShowNotice( true );
+		}
+	}, [ billingStatus, showNotice ] );
+
 	if ( ! hasFinishedResolution ) {
 		return <SpinnerCard />;
 	}
 
+	if ( billingStatus.status === APPROVED && ! showNotice ) {
```

Comment:
Nit: This could be combined with the if below and use a tertiary to show the success component or `null` based on the value of `showNotice`, which could improve readability and make the relationship between these two logic branches more obvious. Not a blocker.

Reply:
Nice. Updated!